src/parameters/Parameters.jl

```diff
@@ -85,4 +59,11 @@ detr_coeff(ps::ACAP) = ps.detr_coeff
 day(ps::ACAP) = IP.day(insolation_params(ps))
 tot_solar_irrad(ps::ACAP) = IP.tot_solar_irrad(insolation_params(ps))
 
+# Define parameters as functions
+# Place derived parameters in exceptions vector
+exceptions = []
+for var in filter(x -> !(x in exceptions), fieldnames(ClimaAtmosParameters))
```

Comment:
```suggestion
for var in fieldnames(ClimaAtmosParameters)
```
I'd recommend removing the exceptions part

Reply:
(sorry I'm late to review this)

Reply:
Oops, I can add this in #2112 or as a standalone PR